functional_tests/scylla_operator/libs/helpers.py

```diff
@@ -32,3 +33,33 @@ def scylla_services_names(db_cluster):
     services = db_cluster.k8s_cluster.kubectl(f"get svc -n {SCYLLA_NAMESPACE} -l scylla/cluster=sct-cluster "
                                               f"-o=custom-columns='NAME:.metadata.name'")
     return [name for name in services.stdout.split() if name not in ('NAME', 'sct-cluster-client')]
+
+
+def rollout_restart(db_cluster: ScyllaPodCluster, namespace: str, deployment: str = None):
+    db_cluster.k8s_cluster.kubectl(f"rollout restart deployment -n {namespace} {deployment if deployment else ''}")
+
+
+def scylla_operator_rollout_restart(db_cluster: ScyllaPodCluster):
+    rollout_restart(db_cluster, namespace=SCYLLA_OPERATOR_NAMESPACE)
+
+
+def get_deployment_rollout_status(db_cluster: ScyllaPodCluster, namespace: str, deployment: str, timeout: str):
+    return db_cluster.k8s_cluster.kubectl(f"rollout status deployment -n {namespace} {deployment} --timeout={timeout}")
+
+
+def wait_for_scylla_operator_rollout_complete(db_cluster: ScyllaPodCluster, timeout: str = '60s'):
+    status = []
+    for deployment in ['scylla-operator', 'webhook-server']:
+        result = get_deployment_rollout_status(db_cluster, namespace=SCYLLA_OPERATOR_NAMESPACE,
+                                               deployment=deployment, timeout=timeout)
+        if f"deployment \"{deployment}\" successfully rolled out" not in result.stdout:
+            status.append(f"{deployment}: {result.stdout}")
+    return status
+
+
+def scylla_operator_pods_and_statuses(db_cluster):
+    pods = db_cluster.k8s_cluster.kubectl(f"get pods -n {SCYLLA_OPERATOR_NAMESPACE} "
+                                          f"-o=custom-columns='NAME:.metadata.name,STATUS:.status.phase'")
+
+    return [name.split() for name in pods.stdout.split('\n') if
+            'NAME' not in name and 'scylla-operator' in name]
```

Comment:
This is wrong way for filtering pods. Use labels for picking up proper pods.

Reply:
done